open_moreinfo.module

```diff
@@ -248,24 +265,22 @@ function open_moreinfo_get_objects($object_id, $object_type = NULL) {
 
 /**
  * Helper function to fetch and save a cover image file.
- *
- * @param string $filename
- *    File name, including its path within Drupal's file folder.
- * @param string $image_url
- *    URL for the source image file.
- *
- * @return mixed
- *    A file object or FALSE on error.
  * @see image_style_create_derivative()
  *
+ * @param $object_id
+ * @param $object_type
+ * @param $filetype
+ * @param false|string $object_url
+ *
+ * @return false|\stdClass|string
  */
 function _open_moreinfo_fetch_object($object_id, $object_type, $filetype, $object_url = FALSE) {
   if (!$object_url) {
     return FALSE;
   }
 
   $filename =
-    file_default_scheme() . '://moreinfo/' . $object_type . '/' . md5($object_id) . $filetype;
+    file_default_scheme() . '://moreinfo/' . $object_type . '/' . md5($object_id) . md5($object_url) . $filetype;
```

Comment:
Hvad er $object_url, og hvornår er den ikke FALSE?

Reply:
object_url er f.eks.: 
https://moreinfo.addi.dk/2.11/more_info_get.php?lokalid=47682576&attachment_type=f56_a&bibliotek=870970&source_id=870970&key=ac6d4b03055b8d577b34

Reply:
Jeg er nødt til at tage den med her fordi der er flere filer nu.

Reply:
Ok - kunne bare ikke lige se, hvor du sender den med fra - kigger lige igen....

Reply:
Ah jeg talte en for meget :) og lurede ikke at det er $moreinfo_file/$moreinfo[$faust_id]->$type